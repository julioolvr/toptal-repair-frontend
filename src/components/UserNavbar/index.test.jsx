import React from 'react';
import { shallow } from 'enzyme';

import { UserNavbar } from '.';

it('renders a navbar', () => {
  const user = { email: 'test@test.com' };
  const wrapper = shallow(<UserNavbar user={user} />);
  expect(wrapper).toMatchSnapshot();
});
