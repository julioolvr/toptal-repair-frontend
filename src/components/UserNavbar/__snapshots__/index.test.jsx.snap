// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a navbar 1`] = `
<div>
  Oh hi 
  test@test.com
</div>
`;
